pkg/engine/renderer/dependencies_test.go

```diff
@@ -55,10 +55,10 @@ func TestGetResources(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			dc := newDependencyCalculator(tt.client, tt.taskObjects)
 			obj, err := dc.resourceDependency(resourceDependency{gvk: typeConfigMap, name: "configmap", namespace: "namespace"})
-			assert.NilError(t, err, "resourceDependency return error")
+			assert.NoError(t, err, "resourceDependency return error")
 			cmResult := &corev1.ConfigMap{}
 			_ = runtime.DefaultUnstructuredConverter.FromUnstructured(obj.UnstructuredContent(), cmResult)
-			assert.DeepEqual(t, cm.Data, cmResult.Data)
+			assert.Equal(t, cm.Data, cmResult.Data)
```

Comment:
is Equal and DeepEqual the same thing?

Reply:
Yes, I checked the source code and it's calling `reflect.DeepEqual` under the hood.